client/src/main/java/client/control/ClientControl.java

```diff
@@ -0,0 +1,31 @@
+package client.control;
+
+import java.io.IOException;
+
+public class ClientControl {
+
+    private final ClientConnection clientConnection;
+
+    public ClientControl() throws IOException {
+        this.clientConnection = new ClientConnection();
+        clientConnection.startConnection();
```

Comment:
move clientConnection.startConnection() to separate ClientControl.start() method